src/todolist.test.js

```diff
@@ -1,38 +1,69 @@
-const AddItem = require('./add.js');
-const add = require('./add.js');
+const addNewItem = require('./add.js');
+
+const updateDescription = require('./editChore.js');
 const deleteItem = require('./remove.js');
+const resetAll = require('./removeAll.js');
+const completeTrue = require('./completeTrue.js');
+
 
- describe('initial', () => {
-  test('first tests', () => {
-    expect(true).toBe(true)
-  })
-}) 
 // add testing
 describe('add a chore', () => {
-  test("contains at least one item on the ordered list", () => {
+  test('contains at least one item on the ordered list', () => {
+    if (typeof window !== 'undefined') {
+      const list = document.querySelector('.chore');
+      list.push({
+        ID: 1,
+        chore: 'run code',
+        complete: false,
+      });
```

Comment:
- [ ] Great job in writing `tests` for your code. Unfortunately, none of them passes. Kindly fix them 🙏 

![image](https://user-images.githubusercontent.com/74814780/182003483-20ce0678-dd7d-40b5-b542-8dca9c6fca7b.png)

Reply:
Hello @Ugboaja-Uchechi , that´s weird. they do pass at my end
![image](https://user-images.githubusercontent.com/103516637/182003635-10f40358-6a75-4a7b-a560-ca0432ea77ba.png)